ProcessQueue/__init__.py

```diff
@@ -33,46 +33,34 @@ def process_scan_report_sheet_table(sheet):
     results = []
     # Get max number of columns in the sheet
     max_column = sheet.max_column
-    # Set a column index (openpyxl index at 1 instead of 0)
-
+    # Skip headers, set min_row & row_idx=2
     for row_idx, row_cell in enumerate(
         sheet.iter_rows(min_row=2, max_col=max_column), start=2
     ):
 
         # Works through pairs of value/frequency columns
         for column_idx, cell in enumerate(row_cell):
-            freq = sheet.cell(row=row_idx, column=column_idx + 2).value
-            if cell.value:
-                if (column_idx) % 2 == 0:
-                    # As we move down rows, checks that there's data there
-                    # This is required b/c value/frequency col pairs differ
-                    # in the number of rows
-                    if (
-                        sheet.cell(row=row_idx, column=column_idx + 1).value == ""
-                        and sheet.cell(row=row_idx, column=column_idx + 2).value == ""
-                    ):
-                        continue
-
-                    # Append to Results as (Field Name,Value,Frequency)
-                    results.append(
-                        (
-                            str(sheet.cell(row=1, column=column_idx + 1).value),
-                            str(sheet.cell(row=row_idx, column=column_idx + 1).value),
-                            sheet.cell(row=row_idx, column=column_idx + 2).value,
-                        )
+            if (column_idx) % 2 == 0:

```

Comment:
One brief edit please before this can be merged - could you change l42 to `enumerate(row_cell, start=1)` and then here `if (column_idx) % 2 == 1` and then below reduce `column_idx+1` to `column_idx` and `column_idx+2` to `column_idx+1`.

This just makes it clearer that we're using a 1-based index as `sheet.cell` expects.